frontend/src/plugins/i18n.js

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue'
+import VueI18n from 'vue-i18n'
+
+Vue.use(VueI18n)
+
+function loadLocaleMessages () {
+  const locales = require.context('@/locales', true, /[A-Za-z0-9-_,\s]+\.json$/i)
+  const messages = {}
+  locales.keys().forEach(key => {
+    const matched = key.match(/([A-Za-z0-9-_]+)\./i)
+    if (matched && matched.length > 1) {
+      const locale = matched[1]
+      messages[locale] = locales(key)
+    }
+  })
+  return messages
+}
```

Comment:
Scheint mir etwas aufwändige Logik nur um alle Files aus einem Ordner zu laden in dem praktisch nie neue Dateien hinzu kommen. Ich bin ja meistens für abstrakte, flexible Lösungen, aber wäre das hier nicht simpler zu verstehen?

```suggestion
import en from '@/locales/en.json'
import de from '@/locales/de.json'

function loadLocaleMessages () {
  return { en, de }
}
```
Quelle: https://github.com/kazupon/vue-i18n/issues/474#issuecomment-444817451

Reply:
Der Code kommt automatisch mit dem Plugin mit. Aber ja, könnte man natürlich vereinfachen :-)

Reply:
@carlobeltrame  So wird nicht mehr lazyloaded oder?